packages/invoices/src/InboundShipment/DetailView/modals/utils.ts

```diff
@@ -28,6 +28,7 @@ const createDraftInboundLine = ({
     packSize: 1,
     isCreated: seed ? false : true,
     expiryDate: undefined,
+    location: undefined,
```

Comment:
another from #1053

Reply:
Thanks - this is for inbound location tab having a whinge